pkgs/development/python-modules/coapthon3/default.nix

```
@@ -0,0 +1,22 @@
+{ fetchFromGitHub
+, lib
+, pythonPackages }:
```

`fetchFromGitHub` is unused. Will remove.
```suggestion
{ lib
, pythonPackages }:
```